### PR TITLE
Mogith pn patch 1

### PR DESCRIPTION
**Pull request by mogith-pn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,13 @@ A repo consists of links for the awesome clarifai modules
...
+| Module DocQA | [![GitHub](https://img.shields.io/badge/GitHub-Link-blue?logo=github)](https://github.com/Clarifai/module-docQA)| This Module showcase the usefullness of Retrieval Augmented Text Generation where User can upload a document and perform variety of tasks. |
+| Module Gallery | [![GitHub](https://img.shields.io/badge/GitHub-Link-blue?logo=github)](https://github.com/Clarifai/module-gallery) | Module-gallery acts as a template for the users to perform different tasks with inputs (Images & Texts) and to visualize the metrics of the app. |
+| Module Template | [![GitHub](https://img.shields.io/badge/GitHub-Link-blue?logo=github)](https://github.com/Clarifai/module-template) | This is a Module-template repository acts as a basic template to make it easy for the users to get started with creating a custom UI module with Clarifai. |
+| Module Prompting | [![GitHub](https://img.shields.io/badge/GitHub-Link-blue?logo=github)](https://github.com/Clarifai/module-prompting) | Base repo from which LLM battleground is developed. |
 
 
 ## Private Modules
```
Comment (sanjaychelliah) [inline README.md:16]: The heading of second column is `github url`. But these are Clarifai URLs

Comment (sanjaychelliah) [reply README.md:16]: Then can we change the heading of the URL column to `Clarifai URL` or something relevant for private modules table?

Comment (mogith-pn) [reply README.md:16]: sure. Changed !

**Review by sanjaychelliah — APPROVED** [review]
LGTM!

**Pull request merged**